calblueprint/unloop
# Create from scratch action item creation now uploads date

Previously, dates weren't being uploaded for newly created Action Items. This PR fixes that. Now, assignments created from scratch will upload with date attached.

This PR does not handle Action Items created from the "Add From Existing" option. There will be a separate PR that will do that.

## To Test:
Run the app and create an Action Item from scratch with a due date. When that date is created, check in the database that the newly created action item has the appropriate date uploaded. Try this for both assignment creation modes (IE multiple action items being created for a single participant and a single action item being created for multiple participants)

CC: @didvi
